pkgs/top-level/aliases.nix

```
@@ -833,6 +833,7 @@ mapAliases ({
   matrique = spectral; # Added 2020-01-27
   maui-nota = libsForQt5.mauiPackages.nota; # added 2022-05-17
   mcgrid = throw "mcgrid has been removed from nixpkgs, as it's not compatible with rivet 3"; # Added 2020-05-23
+  mcomix3 = mcomix; # Added 2022-06-05
```

If I understand correctly, this is the case for a `throw`.

```suggestion
  mcomix3 = throw "mcomix3 is now deprecated in favor of original mcomix"; # Added 2022-06-05
```